app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionFragment.kt

```diff
@@ -93,7 +93,7 @@ class TransactionDetailsActionFragment : BaseViewBindingFragment<FragmentTransac
 
         address?.let {
             with(binding) {
-                content.addView(getTransferItem(it, amount ?: "", addressInfoIndex?.get(it.asEthereumAddressChecksumString())))
```

Comment:
Why do we pass the chain here as a parameter, when it is already a property in TransactionDetailsActionFragment?